### PR TITLE
Add tetunori libraries

### PR DESCRIPTION
**Pull request by tetunori**
Add following 3 libs:
p5.toio, bmwalker.js, p5.simpleAR

### CODE REVIEW

File: libraries/p5.simpleAR.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: p5.simpleAR
+description: A simple JavaScript library that easily converts existing p5.js sketches to AR.
+category: 3d
+sourceUrl: https://github.com/tetunori/p5.simpleAR
+featuredImage:
+  url: p5.simpleAR.png
+  altText: A logo and description of 'p5.simpleAR' library are placed in the center.
+  name: Tetsunori Nakayama
```
Comment (davepagurek) [inline libraries/p5.simpleAR.yaml:9]: Looks like the `author:` key in this and in `p5.toio.yaml` get taken out, could I get you to add those back? Thanks!

Comment (tetunori) [reply libraries/p5.simpleAR.yaml:9]: @davepagurek Thanks for your checking!! 
I seemed to delete `author` key. I will add them later!

Comment (tetunori) [reply libraries/p5.simpleAR.yaml:9]: @davepagurek Since I have added `author` keys, could you kindly check them again?

Comment (davepagurek) [reply libraries/p5.simpleAR.yaml:9]: Looks good, thanks!

**Review by davepagurek — APPROVED** [review]

**Pull request merged**